DEVELOPMENT.md

```diff
@@ -251,38 +245,53 @@ The folder structure of the frontend looks as follows:
 ├── template.go       go template module used to render the frontend HTML
 ```
 
+#### Interactive development stack launcher tool
+
+In order to easily launch development environments in different deployment contexts, this mage target configures and starts a development environment for The Things Stack, allowing users to choose between local and staging environments, enable branding, and configure cloud-hosted mock setups. You can launch it via:
+
+```bash
+$ tools/bin/mage dev:serveDevWebui
+```
+
+It will interactively guide you through the desired setup and launches the The Things Stack Enterprise as well as a frontend development server wia webpack.
```

Comment:
```suggestion
It will interactively guide you through the desired setup and launches the The Things Stack as well as a frontend development server wia webpack.
```